esiclient/tests/unit/v1/test_node_console.py

```diff
@@ -0,0 +1,70 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+from osc_lib import exceptions
+
+from esiclient.tests.unit import base
+from esiclient.v1 import node_console
+
+
+class TestNodeConsoleConnect(base.TestCommand):
+
+    def setUp(self):
+        super(TestNodeConsoleConnect, self).setUp()
+        self.cmd = node_console.NodeConsoleConnect(self.app, None)
+
+        self.node1 = {
+            "uuid": "11111111-2222-3333-4444-aaaaaaaaaaaa",
+            "name": "node1",
+            "console_enabled": False,
+            "console_info": None
+        }
+        self.node2 = {
+            "uuid": "11111111-2222-3333-4444-bbbbbbbbbbbb",
+            "name": "node2",
+            "console_enabled": True,
+            "console_info": {"type": "socat", "url": "tcp://10.10.10.10:1234"}
+            }
+        self.node3 = None
+
+    def test_take_action(self):
```

Comment:
It looks like there are three separate test cases in here; you'll want to split them up, similarly to https://github.com/CCI-MOC/python-esiclient/blob/master/esiclient/tests/unit/v1/test_node_network.py#L127-L235, so that it's easy to see what's being tested with each result.